### DISCUSSION TITLE
PAPI-API bug... I think?

### DISCUSSION THREAD

**Discussion by klaidoshka** [PlaceholderAPI/PlaceholderAPI#837] [(2) Support] [answered]
Expansion registering and unregistering in LocalExpansionManager doesn't work. For as much as I saw, I am somewhat certain that it is because of bad casing. 🥴

#register
```
final String identifier = expansion.getIdentifier().toLowerCase();
...
```
#unregister
```
if (expansions.remove(expansion.getIdentifier()) == null) { return false; }
...
```

I am always getting false in #unregister... If that is not because of the stuff I mentioned above, I am looking for some help then... to register and re-register my own expansion after plugin's reload.

**Comment by Andre601** [comment, marked as answer]
https://github.com/PlaceholderAPI/PlaceholderAPI/pull/793
